test/functional/p2p_filter.py

```diff
@@ -135,14 +136,22 @@ def test_msg_mempool(self):
         self.log.info("Check that a node with bloom filters enabled services p2p mempool messages")
         filter_peer = P2PBloomFilter()
 
-        self.log.debug("Create a tx relevant to the peer before connecting")
-        txid = self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=filter_peer.watch_script_pubkey, amount=9 * COIN)["txid"]
+        self.log.info("Create two tx before connecting, one relevant to the node another that is not")
+        rel_txid = self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=filter_peer.watch_script_pubkey, amount=1 * COIN)["txid"]
+        irr_txid = self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=getnewdestination()[1], amount=2 * COIN)["txid"]
+        irr_wtxid = self.nodes[0].getmempoolentry(irr_txid)["wtxid"]
```

Comment:
nit (feel free to ignore): MiniWallet's `send_to` already returns both txid and wtxid, so one doesn't need the extra `getmempoolentry` RPC call.

Reply:
I'm taking this